scripts/qa_crowbarsetup.sh

```diff
@@ -4754,8 +4754,45 @@ function onadmin_rebootneutron
     onneutron_wait_for_neutron
 }
 
-# This will adapt Cloud6 admin server repositories to Cloud7 ones
-function onadmin_prepare_cloudupgrade_repos_6_to_7
+function onadmin_upgrade_ses_to_4
+{
+    # 1. move ceph nodes to crowbar_upgrade state
+    crowbar_api_request POST $crowbar_api /utils/ceph_pre_upgrade/prepare.json
+
+    # 2. find ceph nodes
+    local ceph_nodes=$(knife search node "roles:ceph-*" -a name | \
+        grep ^name: | cut -d : -f 2 | sed 's/\s//g')
+
+    # 2b save the search for ceph-mons
+    local ceph_mons=$(knife search node "roles:ceph-mon" -a name | \
```

Comment:
This search does not work at this time, I'll have to move it before the preparation...